README.md

```diff
@@ -178,7 +178,7 @@ docker build -t <registry>/<your-username>/console-operator:<version> .
 # following: docker.io/openshift/origin-console-operator:latest
 # for development, you are going to push to an alternate registry.
 # specifically it can look something like this:
-docker build -f Dockerfile.rhel7 -t quay.io/benjaminapetersen/console-operator:latest .
+docker build -f Dockerfile.ocp -t quay.io/benjaminapetersen/console-operator:latest .
```

Comment:
@Leo6Leo Would it be clearer to replace the `benjaminapetersen` with a placeholder like `<your-username>`. Also, this appears in multiple places in the file.